generate-relayerhub.js

```diff
@@ -23,6 +23,7 @@ program.parse(process.argv);
 
 const data = {
   mock: program.mock,
+  network: program.network,
```

Comment:
We also should delete `network` from `generate-system.js` as it is no longer used there.